### PR TITLE
Support DefaultRipple flag (RLJS-280)

### PR DESCRIPTION
**Pull request by lumberj**

### CODE REVIEW

File: api/settings.js [hunk 1]
```diff
@@ -19,7 +19,8 @@ const AccountRootFlags = {
   DisallowXRP:     { name:  'disallow_xrp', value: 0x00080000 },
   DisableMaster:   { name:  'disable_master', value: 0x00100000 },
   NoFreeze:        { name:  'no_freeze', value: 0x00200000 },
-  GlobalFreeze:    { name:  'global_freeze', value: 0x00400000 }
+  GlobalFreeze:    { name:  'global_freeze', value: 0x00400000 },
+  DefaultRipple:   { name:  'default_ripple', value: ripple.Remote.flags.account_root.DefaultRipple }
```
Comment (mDuo13) [inline api/settings.js]: Why is this different from the other ones?

Comment (lumberj) [inline api/settings.js]: What is different? Where the value is coming from? It's already defined as a constant in ripple-lib, so I'm using that value.

Comment (mDuo13) [inline api/settings.js]: Yeah, are GlobalFreeze, NoFreeze, etc. not also defined as constants in ripple-lib?

Comment (lumberj) [inline api/settings.js]: They are, but I wanted to introduce the minimum set of changes for this

**Comment by mDuo13** [comment]
LGTM (but I am not an expert)

File: api/settings.js (hunk 1 above)
Comment (lumberj) [inline api/settings.js]: I just made the changes you suggested, and found that two of the flags were missing from ripple-lib https://github.com/ripple/ripple-lib/pull/314

**Comment by clark800** [comment]
LGTM

**Pull request merged**